libflux/src/libstd/build.rs

```diff
@@ -0,0 +1,24 @@
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+fn prelude() -> Vec<u8> {
+    Vec::new()
+}
+
+fn stdlib() -> Vec<u8> {
+    Vec::new()
```

Comment:
So I guess it's in these two functions that type inference of the stdlib is going to occur?

Reply:
Right.